sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusReceivedMessage.cs

```diff
@@ -425,7 +425,7 @@ public ServiceBusMessageState State
                     return (ServiceBusMessageState)val;
                 }
 
-                return default;
+                return ServiceBusMessageState.Active;
```

Comment:
Interesting;  as the `0` member, shouldn't this have been `default`?   I agree the explicit return is more clear; I'm just curious.

Reply:
Yes, this is just for the sake of clarity - no functional change.